hack/setup/prepare-crc.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#This script configures CRC for running app-studio in it
+#It cofigures the CRC to have better memory and CPUs 
+#It also configures CRC to have required addons.
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo $ROOT
+CRCBINARY=$(readlink -f ~/.crc/bin/crc)
+
```

Comment:
The CRCBINARY line doesn't work on MacOS, but if you add this line right after, it works on MacOS (it just uses the pre-configured crc command in case readlink doesn't work):
[[ -z "$CRCBINARY" ]] && CRCBINARY="crc"

Definitely want to fix that before merging.

Reply:
This is fixed in the latest commit, thanks.